portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,125 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body{
+  margin:auto;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.landing-section{
+  background: paleturquoise;
+  height: 100vh;
+  width: 100vw;
+  flex-direction: column;
+  align-items: center;
+  display: flex;
+}
+
+#title{
+  position: absolute;
+  margin-top: 41vh;
+  font-size: 56pt;
+  margin-left: 0.75vw;
+  font-family: 'Montserrat', serif;
+}
+
+#tri{
+  width: 35vw;
+}
+
+.social-links{
+  display: flex;
+  justify-content: space-evenly;
+  width: 20vw;
+}
+
+a > img{
+  width: 2vw;
+}
+
+#biography{
+  padding-top: 5vh;
+  width: 50vw;
+  font-size: 24pt;
+  font-family: "Open Sans", serif;
+}
```

Comment:
It's uncommon to style a `p` tag like this - is there a more semantic way we can name this block, or put it inside a parent container with this as a class instead?

For context - paragraph tags are by nature in-line elements, which we don't tend to set widths on.  We generally style their containers, and these will take on the properties of their parent.

I don't anticipate that you will have more than one biography section on this page, but I'd encourage you to think about if is there something else that this belongs to - maybe all text on the page should be in a container that have these properties? Maybe we could set font properties on `body` instead, and customize it where needed, i.e. the subtitle fields you have below.

Reply:
How exactly would <p> tags inherit font if we put it on <body>? I'm under the assumption that it only inherits from direct parents.

Reply:
It inherits from the closest parent with a set style, so if you set it on body and don't override anywhere else, it'll work!